packages/server/utils/updateGitHubSchema.ts

```diff
@@ -4,12 +4,12 @@ import fetch from 'node-fetch'
 
 const updateGitHubSchema = async () => {
   const schemaPath = './githubSchema.graphql'
-  const schemaJSONPath = './githubSchema.json'
+  const schemaJSONPath = './githubSchema.ts'
```

Comment:
i think we can delete this entire file, too. this was for the old way of doing things